Library/Formula/arangodb.rb

```diff
@@ -14,9 +14,17 @@ class Arangodb < Formula
 
   depends_on "go" => :build
   depends_on "openssl"
+  depends_on "readline"
 
   needs :cxx11
 
+  patch :DATA
```

Comment:
It has already been submitted and should be available in the next release (ArangoDB 2.7.1), I will add a comment